lessons/01_phugoid/Rocket_Assignment.ipynb

```diff
@@ -90,109 +90,38 @@
    "source": [
     "In case you are kind of confused about the rocket equations, here we show how to get to them. \n",
     "\n",
-    "We will start from the widely used [Reynolds' transport theorem](http://en.wikipedia.org/wiki/Reynolds_transport_theorem). For an extensive property $N$ in a volume $\\Omega$, with corresponding intensive property $\\eta = N/m$, the Reynolds' transport theorem reads\n",
+    "Newton's second law states that the acceleration of our vehicule times it mass is equal to all the forces acting on it. Therefore,\n",
```

Comment:
I would change this to 

"Newton's second law states that the acceleration of the vehicle times its mass is equal to all the forces acting on it."